command/agent/auth/jwt/jwt.go

```diff
@@ -73,14 +75,32 @@ func NewJWTAuthMethod(conf *auth.AuthConfig) (auth.AuthMethod, error) {
 		return nil, errors.New("could not convert 'role' config value to string")
 	}
 
+	if removeJWTAfterReadingRaw, ok := conf.Config["remove_jwt_after_reading"]; ok {
+		removeJWTAfterReading, err := parseutil.ParseBool(removeJWTAfterReadingRaw)
+		if err != nil {
+			return nil, fmt.Errorf("error parsing 'remove_jwt_after_reading' value: %w", err)
+		}
+		j.removeJWTAfterReading = removeJWTAfterReading
+	} else {
+		// Default to true for maximum backwards compatibility.
+		j.removeJWTAfterReading = true
```

Comment:
It looks like this is defaulting to `true` in the initial declaration of `j`. Do we need it both places?

Reply:
Oops, good spot. I lost that in the formatting diff noise.